src/legacy/components/Profile.tsx

```diff
@@ -0,0 +1,281 @@
+/* eslint-disable */
+import React, { Component } from 'react'
+import styles from './Profile.module.css'
+import classnames from 'classnames'
+import Rate from 'rc-rate'
+import 'rc-rate/assets/index.css'
+import '../assets/css/rc-rate.custom.css'
+
+export default class Profile extends Component<any, any> {
+  render() {
+    // const {  } = this.props
+    return (
+      <div className={classnames('px-8 py-10 w-full flex', styles.profile)}>
+        <div
+          className={classnames(
+            'm-auto w-full bg-white px-14 py-8 flex',
+            styles.wrapper,
+          )}
+        >
+          <div className='flex flex-col justify-between'>
+            <Card />
+            <div className='text-gray-400 text-sm'>
+              Member Since May 15, 2021
+            </div>
+          </div>
+          <div className='flex flex-col flex-grow pl-4'>
+            <Tabs />
+            <div className='flex pt-4 pl-2 justify-between'>
+              <General />
+              <Relations />
+            </div>
+          </div>
+        </div>
+      </div>
+    )
+  }
+}
+
+class Card extends Component<any, any> {
+  render() {
+    return (
+      <div
+        className={classnames(
+          'flex flex-col pb-4 rounded-md shadow-medium',
+          styles.card,
+        )}
+      >
+        <div className='bg-blue-300 h-32 rounded-t-md'></div>
+        <div className='-mt-10 px-4 flex'>
+          <div className='rounded-full border-4 border-white bg-pink-200 w-24 h-24'></div>
+          <div className='mt-11 pl-3 text-sm'>
+            <div className='px-1 text-gray-300'>@zndrson</div>
+            <div className='pt-2 text-blue-400'>0xc41923...2145</div>
```

Comment:
This is the user's wallet address, could you implement the real truncate instead of hard coding `...` ?